src/RegExpPattern.php

```diff
@@ -32,7 +35,7 @@ class RegExpPattern
     public static function tables()
     {
         $pattern = '/(?<creationScript>CREATE\s+TABLE\s+`(?<tableName>\S+)`\s+';
-        $pattern .= '\((?<tableDefinition>[^;\/]+)\)';
+        $pattern .= '\((?<tableDefinition>([^;\/]+?(.COMMENT.\'[^\']+((\'\')[^\']*)*\'(?!=\')))+.*?|[^;\/]+?)\)';
```

Comment:
Can you please put the "comment" regular expression in a new line. It will be easier to read that way.